openshift/system/Dockerfile

```diff
@@ -1,82 +1,75 @@
-FROM quay.io/centos7/ruby-27-centos7 AS porta-base
-
-USER root
-
+FROM quay.io/centos/centos:stream8 as builder
+WORKDIR /opt/system
 ENV RUBY_MAJOR_VERSION=2 \
-    RUBY_MINOR_VERSION=7
+    RUBY_MINOR_VERSION=7 \
+    RAILS_ENV=production \
+    NODE_ENV=production
+ENV RUBY_VERSION="${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}"
+ADD . ./
+ADD openshift/system/config/* ./config/
+RUN dnf -y module enable ruby:${RUBY_VERSION} nodejs:16 mysql:8.0 \
+    && dnf install -y --setopt=skip_missing_names_on_install=False,tsflags=nodocs shared-mime-info make automake gcc gcc-c++ redhat-rpm-config postgresql rubygem-irb rubygem-rdoc ruby-devel nodejs libpq-devel mysql-devel gd-devel git 'dnf-command(download)' \
```

Comment:
What's this `'dnf-command(download)' ` about?

Reply:
It installs the proper dnf plugin to implement `dnf download`.